fbgemm_gpu/src/cumem_utils.cu

```diff
@@ -295,6 +324,7 @@ void uvm_cuda_mem_advise(Tensor t, int64_t cudaMemoryAdvise) {
 #endif
```

Comment:
First of all, do you need to create a separate function for this (i.e., can't the hipification script convert the CUDA version)?

If we don't need to, we have to maintain two functions, which would be great.
If we need to create a separate function for AMD GPUs, can you remove the outdated `__HIP_PLATFORM_HCC__` from the CUDA code above?

Reply:
Just a reminder: do you need a separate function for `uvm_cuda_mem_advise()`?

Reply:
Sorry, I overlooked this comment before. 
Currently, we need to keep this `#ifdef __HIP_PLATFORM_HCC__` separated block since `hipify_torch` has some problem of converting `cudaMemoryAdvise`. I'm reaching out to the owner of `hipify_torch`.

Reply:
@liligwu , 
Let get this resolved in `hipify_torch` repo.

Reply:
We've updated the mapping of `cudaMemoryAdvise` in `hipify_torch`. The `#ifdef __HIP_PLATFORM_HCC__` is removed.